tests/ui/missing_assert_message.stderr

```diff
@@ -1,131 +1,113 @@
-error: assert without any message
-  --> $DIR/missing_assert_message.rs:14:5
+error[E0425]: cannot find function `foo` in this scope
```

Comment:
issue: seems like a bug

Reply:
yea, the test was broken before. It would not compile at all with `--test` passed